src/app/sponsors/page.tsx

```diff
@@ -0,0 +1,77 @@
+import Paragraph from '@/components/Paragraph';
+import Sponsors from '@/components/Sponsors';
+import Title from '@/components/Title';
+import type { SponsorType } from '@/data/sponsors';
+import Image from 'next/image';
+
+const SPONSOR_TYPE_COLORS = {
+    gold: '#FCC018',
+    silver: '#C3C3C3',
+    bronze: '#E8903F',
+} as const satisfies Record<SponsorType, string>;
+
+export default function SponsorsPage() {
+    return (
+        <div className="flex flex-col items-center gap-10">
```

Comment:
The margin of the main content should match the home page: `mx-4 md:mx-10`.

Reply:
I don't get this? Why add extra margin? I did `w-responsive` in `layout.tsx`. It's shared for each page.

Reply:
The margins should match the home page since the margins for the home, sponsors and contact page looks like they have different margins. 
<img width="1512" alt="image" src="https://github.com/compsci-adl/website/assets/47909638/b9f6f6ca-40dd-4b8f-b69b-70678f00fd17">
<img width="1512" alt="image" src="https://github.com/compsci-adl/website/assets/47909638/1cf61d94-0baa-423f-a306-be424217ee21">
<img width="1512" alt="image" src="https://github.com/compsci-adl/website/assets/47909638/3bddee6f-c692-4e3d-906b-bf514bcc2b94">

Reply:
ig this is right for sponsor page. You can check figma. It does have different width.

Reply:
the "CODE." on the home page should be capitalised as well.